concent_api/core/payments/bankster.py

```diff
@@ -145,7 +145,7 @@ def claim_deposit(
             # If the total of existing claims and the current claim is greater or equal to the current deposit,
             # we can't add a new claim.
             if provider_deposit <= sum_of_existing_provider_claims['sum_of_existing_claims'] + settings.ADDITIONAL_VERIFICATION_COST:
-                return (requestor_deposit, None)
+                return (None, None)
```

Comment:
This looks like a bugfix for something that was broken before this pull request. It should be a separate commit.

Reply:
I think it is now seen because branch is not rebased. It must `return (requestor_deposit, None)`

Reply:
But why? It's supposed to return a claim instance. `requestor_deposit` is a number.